test/e2e/screenshot.test.js

```diff
@@ -80,29 +103,36 @@ test('Screenshot the home view', async t => {
   });
 });
 
-fixture(`Activity view`).page(`${baseURL}/#/activity/fakedata`);
+fixture(`Activity view`).page(`${baseURL}/#/activity/fakedata`).requestHooks(HTTPLogger);
 
-test('Screenshot the activity view', async t => {
+test.clientScripts({
+  content: logJsErrorCode,
+})('Screenshot the activity view', async t => {
   await hide_devonly(t);
   await waitForLoading(t);
-  await checkNoError(t);
   await t.takeScreenshot({
     path: 'activity.png',
     fullPage: true,
   });
+  await checkNoError(t);
```

Comment:
wow, did not realize this was all I had to do to get screenshots on errors :rofl: 

Very nice!